api_tests/schema_responses/views/test_schema_response_detail.py

```diff
@@ -495,6 +495,13 @@ def test_PATCH_updates_revision_justification(self, app, schema_response, payloa
         schema_response.refresh_from_db()
         assert schema_response.revision_justification == 'why not?'
 
+        payload['data']['attributes']['revision_justification'] = None
```

Comment:
This should be its own test. Also, please add a test for the empty string, since I think that's what caused the issue originally.